packages/template/__tests__/custom-invokable.test.ts

```diff
@@ -68,6 +68,13 @@ invokeBlock(resolve(caseOf)({}, maybeValue), {
   },
 });
 
+// SafeString
+invokeEmit({
```

Comment:
Can you also add a test to the `environment-ember-loose` suite to ensure that the result of an `htmlSafe()` call works?

It should, given how the types are defined today, but since the definitions are totally separate from each other, it would be good to have a check in place to prevent regression if they ever drift in the future.